src/Microsoft.DotNet.Darc/tests/Microsoft.DotNet.Darc.VirtualMonoRepo.E2E.Tests/VmrTestsBase.cs

```diff
@@ -7,16 +7,24 @@
 using System.IO;
 using System.Linq;
 using System.Text;
+using System.Threading;
 using System.Threading.Tasks;
+using System.Web;
 using FluentAssertions;
+using Microsoft.DotNet.Darc.Helpers;
 using Microsoft.DotNet.DarcLib.Helpers;
+using Microsoft.DotNet.DarcLib.VirtualMonoRepo;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
 using NUnit.Framework;
 
 #nullable enable
 namespace Microsoft.DotNet.Darc.Tests.VirtualMonoRepo;
 
 public abstract class VmrTestsBase
 {
+    private Lazy<IServiceProvider> _serviceProvider = null!;
+    private readonly CancellationTokenSource _cancellationToken = new();
```

Comment:
Instead of this, I'd just pass `default` to the Vmr classes which equals to `CancellationToken.None`